togetherly/lib/views/widgets/chore_list.dart

```diff
@@ -0,0 +1,171 @@
+    return Column(
+      children: <Widget>[
+        for (final chore in chores)
+          Container(
+            height: 65,
+            margin: const EdgeInsets.only(top: 5),
+            decoration: BoxDecoration(
+              borderRadius: BorderRadius.circular(5),
+              color: AppColors.brandLightGray,
+            ),
+            child: Padding(
+              padding: const EdgeInsets.only(
+                  top: 6.0, bottom: 6.0, right: 10.0, left: 10.0),
+              child: Row(
+                mainAxisAlignment: MainAxisAlignment.spaceBetween,
+                children: <Widget>[
+                  Row(
+                    children: <Widget>[
+                      if (chore.isBonus)
+                        const Icon(
+                          Icons.star,
+                          color: AppColors.brandGold,
+                        ),
+                      if (chore.isBonus)
+                        const SizedBox(
+                          width: 5,
+                        ),
+                      Column(
+                        mainAxisAlignment: MainAxisAlignment.center,
+                        crossAxisAlignment: CrossAxisAlignment.start,
+                        children: [
+                          Text(
+                            chore.title,
+                            style: chore.status == ChoreStatus.PENDING ||
+                                    chore.status == ChoreStatus.COMPLETED
+                                ? AppTextStyles.brandBodyStrike
+                                : AppTextStyles.brandBody,
+                          ),
+                          if (isDueToday(chore))
+                            Text(
+                              prettyDate(chore.dueDate),
+                              style: AppTextStyles.brandAccentSub,
+                            ),
+                        ],
+                      ),
+                    ],
+                  ),
+                  Row(
+                    children: <Widget>[
+                      const Icon(
+                        Icons.bolt,
+                        size: 32,
+                        color: AppColors.brandGold,
+                      ),
+                      Text(
+                        chore.points.toString(),
+                        style: AppTextStyles.brandAccentLarge,
+                      ),
+                      const SizedBox(
+                        width: 15,
+                      ),
+                      getStatusIcon(chore),
+                    ],
+                  ),
+                ],
+              ),
+            ),
+          ),
+      ],
+    );
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    return Column(
+      mainAxisAlignment: MainAxisAlignment.start, // Aligns to center
+      crossAxisAlignment: CrossAxisAlignment.start, // Aligns to left
+      children: <Widget>[
+        Text(
+          widget.title,
+          style: AppTextStyles.brandAccentLarge,
```

Comment:
Longer term, I would like to leverage [Flutter's built-in theming](https://docs.flutter.dev/cookbook/design/themes) framework rather than manually specify commonly-used text styles for individual widgets. Same goes for other places where you manually select a color or text style from the const values.

Reply:
I did spend an hour or two trying to understand the theming framework. It seemed very strict on what it allowed me to do and forces specific theme names on you for colors and text styles. It made more sense to me to write my own classes for themes and just reference them the way I did. If you understand how to use the Theme framework, let's figure that out together.

Reply:
I think the strictness is intentional, as it is meant to give you a well-defined set of options to configure, like accent color, background color, text styles for different heading levels, etc. Doing the way you have shown is more flexible, but that flexibility also comes with more responsibility to not combine colors or text styles in weird, non-standard ways. Which is perfectly achievable, it just requires more work on your part. The link in my previous comment has some examples. Since you're our UI guy, I'll leave it up to you to decide which method you prefer.

Reply:
I understand. I think I don't mind the flexibility combined with extra work for now because I may not necessarily have definitively chosen what may be the accent color vs other types of colors, and the flexibility in development to say, "oh actually this color works better in this spot" makes more sense to me to just have it named the color rather than trying to apply the Themed accent color to something that doesn't feel like an accent. I don't know if that made sense, but I'll stick with that for now and look more into understanding their Theming system as I have time.